platform/xbox/include/stdatomic.h

```diff
@@ -0,0 +1,150 @@
+/* Atomics <stdatomic.h>
+
+   This file is part of the Public Domain C Library (PDCLib).
+   Permission is granted to use, modify, and / or redistribute at will.
+*/
+
+#ifndef _PDCLIB_STDATOMIC_H
+#define _PDCLIB_STDATOMIC_H _PDCLIB_STDATOMIC_H
+
+#include <pdclib/_PDCLIB_int.h>
+
+#define ATOMIC_BOOL_LOCK_FREE __CLANG_ATOMIC_BOOL_LOCK_FREE
```

Comment:
Given that all of this is specific to clang, shouldn't there at least be some custom `#error Compiler not supported` for cases where clang isn't being used?

With nxdk finally getting the standard-toolchain format (bin/ and include/) ports to other compilers are very likely and people might also port more complicated projects which compile a subset of the application with another non-clang compiler.

Reply:
Agreed. I added a check.